src/signalfx_metadata.py

```diff
@@ -1034,14 +1034,13 @@ def get_aws_info(host_info={}):
     call into aws to get some information about the instance, timeout really
     small for non aws systems and only try the once per startup
     """
-    global AWS
-    if not AWS:
+    global AWS, AWS_SET
+    if AWS_SET and not AWS:
```

Comment:
this is impossible.  you cannot have AWS set to False and AWS_SET be True.  I would remove this return statement completely.  We want to be sending the aws events in.